spec/indico_batch_spec.rb

```diff
@@ -21,9 +22,7 @@
     expected_keys = Set.new(%w(Conservative Green Liberal Libertarian))
     data = ['Guns don\'t kill people.', ' People kill people.']
     response = Indico.batch_political(data,
-                                      @username,
-                                      @password,
-                                      @private_cloud)
+                                      @config_private_cloud )
```

Comment:
Ah, it did when there were more arguments passed. It can be on the same line now. Good catch!